src/components/account-header/account-header.yaml

```diff
@@ -214,17 +211,17 @@ examples:
       account-menu-dropdown-open:
         beforeTakingScreenshot:
           - click: ".hmrc-account-menu__link[aria-expanded='false']"
-          - waitFor: ".main-nav-is-open"
+          - waitFor: 1000
       account-menu-dropdown-open-active-link-hovered:
         beforeTakingScreenshot:
           - click: ".hmrc-account-menu__link[aria-expanded='false']"
-          - waitFor: ".main-nav-is-open"
-          - hover: ".hmrc-account-menu__link[href='/messages/href']"
+          - waitFor: 1000
+          - hover: ".hmrc-account-menu__link[href$='/messages/href']"
       account-menu-dropdown-open-active-link-focused:
         beforeTakingScreenshot:
           - click: ".hmrc-account-menu__link[aria-expanded='false']"
-          - waitFor: ".main-nav-is-open"
-          - focus: ".hmrc-account-menu__link[href='/messages/href']"
+          - waitFor: 1000
```

Comment:
might be being optimistic about 1000ms being long enough wait, but I'm imagining that main-nav-is-open is pretty instantaneous and it's just waitForSelector not matching up with the css applying